solidity-v1/test/TestPhasedEscrow.js

```diff
@@ -810,6 +841,73 @@ describe("BatchedPhasedEscrow", () => {
       )
     })
   })
+
+  describe("when withdrawing to a PlainTransferEscrowBeneficiary", () => {
+    const baseBalance = 200000000
+    const transferAmount = 9100000
+
+    let rewardsContract
+    let rewardsBeneficiary
```

Comment:
```suggestion
    let recipientContract
    let escrowBeneficiary
```

Reply:
Here I can do the rename but I kept it aligned with `assertRewards` earlier. WDYT?